actions/qemuimg2disk/v1/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.13.5 as builder
+ENV QEMU_NAME="qemu"
+ENV QEMU_REV="6.0.0-rc3"
+ENV QEMU_SRC_BASENAME="${QEMU_NAME}-${QEMU_REV}"
+ENV QEMU_SRC_URL="https://download.qemu.org/${QEMU_SRC_BASENAME}.tar.xz"
+
+RUN apk update
+RUN apk upgrade
+
+# required by qemu
+RUN apk add\
+ make\
+ samurai\
+ perl\
+ python3\
+ gcc\
+ libc-dev\
+ pkgconf\
+ linux-headers\
+ glib-dev glib-static\
+ zlib-dev zlib-static
```

Comment:
put a space before the `\` so it matches other lists.
Also, maybe indent a few spaces past `apk`'s column?

Reply:
cleaned up